lib/V8/v8-utils.cpp

```diff
@@ -418,10 +418,12 @@ static void JS_Parse(v8::FunctionCallbackInfo<v8::Value> const& args) {
       v8::Local<v8::Object> exceptionObj =
```

Comment:
TRI_LogV8Exception() also does a direct access to message without check?

Reply:
Well, no:
```
4211 void TRI_LogV8Exception(v8::Isolate* isolate, v8::TryCatch* tryCatch) {
4212   v8::HandleScope handle_scope(isolate);
4213 
4214   TRI_Utf8ValueNFC exception(tryCatch->Exception());
4215   char const* exceptionString = *exception;
4216   v8::Handle<v8::Message> message = tryCatch->Message();
4217 
4218   // V8 didn't provide any extra information about this error; just print the
4219   // exception.
4220   if (message.IsEmpty()) {
```